src/DropDownMenu/DropDownMenu.js

```diff
@@ -28,7 +32,8 @@ function getStyles(props, context) {
     icon: {
       fill: accentColor,
       position: 'absolute',
-      right: spacing.desktopGutterLess,
+      right: isRtl ? '' : iconPadding,
+      left: isRtl ? iconPadding : '',
```

Comment:
You can use [`directionInvariant`](https://github.com/callemall/material-ui/blob/master/src/utils/rtl.js#L12) to disable the auto RTL handling. I think that we should have a look at how react-native handle the issue. I'm sure they have found an elegant solution to the problem. I'm raising that point because we haven't thought about yet for the [`next` branch](https://material-ui-1dab0.firebaseapp.com).